### PR TITLE
bsky: unthemed colors and some fixes

### PR DESCRIPTION
**Pull request by teidesu**
## 🔧 What does this fix? 🔧

- inappropriate hover in latte
- invisible text in latte
- icon contrast fix in mobile layout
- unthemed skeletons and thread lines

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [x] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

File: styles/bsky/catppuccin.user.css [hunk 1]
```diff
@@ -2,7 +2,7 @@
 @name Bluesky Social Catppuccin
 @namespace github.com/catppuccin/userstyles/styles/bsky
 @homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/bsky
-@version 0.0.9
```
Comment (RoootTheFox) [inline styles/bsky/catppuccin.user.css:5]: I'd prefer if this was changed to 0.1.0 instead of 0.0.10

File: styles/bsky/catppuccin.user.css [hunk 2]
```diff
@@ -565,6 +578,26 @@
     [style*="background-color: rgb(185, 185, 193)"] {
       background-color: @overlay2 !important;
     }
+    
+    /* skeletons */
+    .r-kdyh1x, .r-cpet4d {
```
Comment (RoootTheFox) [inline styles/bsky/catppuccin.user.css:583]: are these classes the only way to reliably match skeletons? i've seen these classes change even from small changes, so i'd prefer if there was a more reliable way to match these
(if not then thats okay, we can just update the classes when they (inevitably) break like i do with most of the inline colors already anyways)

**Review by RoootTheFox — CHANGES_REQUESTED** [review]
looks good, does fix the latte issues
i have some very minor issues with it (such as the version being `0.0.10`) but looks good apart from that!

File: styles/bsky/catppuccin.user.css (hunk 2 above)
Comment (teidesu) [reply styles/bsky/catppuccin.user.css:583]: havent found any really :c

their color is the same as "secondary background color" which is already overridden above but looks pretty bad:
image

these classes set the "border-radius" values on the skeletons which *seem* to only be used on them, so combined with the color this should be safe enough

File: styles/bsky/catppuccin.user.css (hunk 1 above)
Comment (teidesu) [reply styles/bsky/catppuccin.user.css:5]: i have no idea how versioning is supposed to work in userstyles tbh 🙃

File: styles/bsky/catppuccin.user.css (hunk 2 above)
Comment (RoootTheFox) [reply styles/bsky/catppuccin.user.css:583]: alright ty !!

**Review by RoootTheFox — APPROVED** [review]
looks good!

**Comment by sgoudham** [comment]
I don't use stylus so I don't know if there are any constraints on versioning from that ecosystem but I think it's worth considering again. 

From what I know at the minute, I think datetime versioning makes a lot of sense here (which can also be automatically incremented as part of CI)

**Comment by uncenter** [comment]
> hm, thats fair - i prefer increasing the minor version when patch overflows (9 -> 10)
> 
> maybe consistent versioning scheme guidelines for userstyles are a thing worth considering?

This isn't how we do it. We follow the regular, traditional guidelines for semantic versioning; fixes are patch bumps, features are minor bumps, breaking changes are major bumps.

**Comment by uncenter** [comment]
The commit title here is also incorrect - it doesn't have a commit type. `bsky: unthemed colors and some fixes` needs to be `fix(bsky): misc`.

**Comment by teidesu** [comment]
sorry >_<

**Pull request merged**